2022-11-15 15:56:49,941 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Configure stats pid to 1637445
2022-11-15 15:56:49,942 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from /nfs/homedirs/zhz/.config/wandb/settings
2022-11-15 15:56:49,944 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/settings
2022-11-15 15:56:49,946 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-11-15 15:56:49,946 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-11-15 15:56:49,947 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Applying login settings: {'api_key': '***REDACTED***'}
2022-11-15 15:56:49,949 INFO    MainThread:1637445 [wandb_init.py:_log_setup():476] Logging user logs to /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/run-20221115_155649-2xacs4dm/logs/debug.log
2022-11-15 15:56:49,950 INFO    MainThread:1637445 [wandb_init.py:_log_setup():477] Logging internal logs to /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/run-20221115_155649-2xacs4dm/logs/debug-internal.log
2022-11-15 15:56:49,951 INFO    MainThread:1637445 [wandb_init.py:_jupyter_setup():426] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f1738f97550>
2022-11-15 15:56:49,954 INFO    MainThread:1637445 [wandb_init.py:init():516] calling init triggers
2022-11-15 15:56:49,955 INFO    MainThread:1637445 [wandb_init.py:init():520] wandb.init called with sweep_config: {}
config: {}
2022-11-15 15:56:49,956 INFO    MainThread:1637445 [wandb_init.py:init():536] re-initializing run, found existing run on stack: 2zehmhc2
2022-11-15 15:56:49,960 INFO    MainThread:1637445 [wandb_run.py:_finish():1746] finishing run uncertainty_tum/test-project/2zehmhc2
2022-11-15 15:56:49,961 INFO    MainThread:1637445 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2022-11-15 15:56:49,963 INFO    MainThread:1637445 [wandb_run.py:_restore():1932] restore
2022-11-15 15:56:49,964 INFO    MainThread:1637445 [wandb_run.py:_restore():1938] restore done
2022-11-15 15:56:55,045 INFO    MainThread:1637445 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2022-11-15 15:56:55,046 INFO    MainThread:1637445 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2022-11-15 15:56:55,058 INFO    MainThread:1637445 [wandb_run.py:_footer_sync_info():3333] logging synced files
2022-11-15 15:56:55,066 INFO    MainThread:1637445 [wandb_init.py:init():569] starting backend
2022-11-15 15:56:55,067 INFO    MainThread:1637445 [wandb_init.py:init():573] setting up manager
2022-11-15 15:56:55,076 INFO    MainThread:1637445 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-11-15 15:56:55,085 INFO    MainThread:1637445 [wandb_init.py:init():580] backend started and connected
2022-11-15 15:56:55,115 INFO    MainThread:1637445 [wandb_run.py:_label_probe_notebook():1113] probe notebook
2022-11-15 15:56:55,115 INFO    MainThread:1637445 [wandb_run.py:_label_probe_notebook():1123] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-11-15 15:56:55,117 INFO    MainThread:1637445 [wandb_init.py:init():658] updated telemetry
2022-11-15 15:56:55,185 INFO    MainThread:1637445 [wandb_init.py:init():694] communicating run to backend with 60 second timeout
2022-11-15 15:56:56,513 INFO    MainThread:1637445 [wandb_run.py:_on_init():2000] communicating current version
2022-11-15 15:56:56,636 INFO    MainThread:1637445 [wandb_run.py:_on_init():2004] got version response 
2022-11-15 15:56:56,636 INFO    MainThread:1637445 [wandb_init.py:init():728] starting run threads in backend
2022-11-15 15:56:59,985 INFO    MainThread:1637445 [wandb_run.py:_console_start():1980] atexit reg
2022-11-15 15:56:59,985 INFO    MainThread:1637445 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.WRAP_RAW
2022-11-15 15:56:59,987 INFO    MainThread:1637445 [wandb_run.py:_redirect():1903] Wrapping output streams.
2022-11-15 15:56:59,987 INFO    MainThread:1637445 [wandb_run.py:_redirect():1925] Redirects installed.
2022-11-15 15:56:59,988 INFO    MainThread:1637445 [wandb_init.py:init():765] run started, returning control to user process
2022-11-15 15:57:00,477 INFO    MainThread:1637445 [wandb_init.py:_pause_backend():389] pausing backend
2022-11-15 15:57:00,477 INFO    MainThread:1637445 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-11-15 16:49:47,186 INFO    MainThread:1637445 [wandb_init.py:_resume_backend():398] resuming backend
2022-11-15 16:49:47,258 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Configure stats pid to 1637445
2022-11-15 16:49:47,258 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from /nfs/homedirs/zhz/.config/wandb/settings
2022-11-15 16:49:47,260 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/settings
2022-11-15 16:49:47,262 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-11-15 16:49:47,263 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-11-15 16:49:47,265 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Applying login settings: {'api_key': '***REDACTED***'}
2022-11-15 16:49:47,265 INFO    MainThread:1637445 [wandb_init.py:_log_setup():476] Logging user logs to /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/run-20221115_164947-360bszfx/logs/debug.log
2022-11-15 16:49:47,267 INFO    MainThread:1637445 [wandb_init.py:_log_setup():477] Logging internal logs to /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/run-20221115_164947-360bszfx/logs/debug-internal.log
2022-11-15 16:49:47,269 INFO    MainThread:1637445 [wandb_init.py:init():516] calling init triggers
2022-11-15 16:49:47,271 INFO    MainThread:1637445 [wandb_init.py:init():520] wandb.init called with sweep_config: {}
config: {}
2022-11-15 16:49:47,271 INFO    MainThread:1637445 [wandb_init.py:init():536] re-initializing run, found existing run on stack: 2xacs4dm
2022-11-15 16:49:47,276 INFO    MainThread:1637445 [wandb_run.py:_finish():1746] finishing run uncertainty_tum/test-project/2xacs4dm
2022-11-15 16:49:47,276 INFO    MainThread:1637445 [jupyter.py:save_history():448] not saving jupyter history
2022-11-15 16:49:47,278 INFO    MainThread:1637445 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-11-15 16:49:47,280 INFO    MainThread:1637445 [wandb_init.py:_jupyter_teardown():408] cleaning up jupyter logic
2022-11-15 16:49:47,282 INFO    MainThread:1637445 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2022-11-15 16:49:47,284 INFO    MainThread:1637445 [wandb_run.py:_restore():1932] restore
2022-11-15 16:49:47,286 INFO    MainThread:1637445 [wandb_run.py:_restore():1938] restore done
2022-11-15 16:49:51,346 INFO    MainThread:1637445 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2022-11-15 16:49:51,346 INFO    MainThread:1637445 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2022-11-15 16:49:51,356 INFO    MainThread:1637445 [wandb_run.py:_footer_sync_info():3333] logging synced files
